Dockerfile

```
@@ -0,0 +1,11 @@
+# This is a development Dockerfile. For versioned Dockerfiles see:
+# https://github.com/praekeltfoundation/docker-seed
+FROM praekeltfoundation/django-bootstrap:py2.7-jessie
+
+ COPY . /app
```

spacing before `COPY`
```suggestion
COPY . /app
```